gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java

```diff
@@ -148,6 +148,7 @@ static Map<String, String> getCustomTags(Map<String, String> customTags,
     ImmutableMap.Builder<String, String> tagsBuilder = ImmutableMap.builder();
     String sourceProjectId = locationTags.get("project_id");
     if (!Strings.isNullOrEmpty(destinationProjectId)
+        && !Strings.isNullOrEmpty(sourceProjectId)
```

Comment:
This change is unrelated to the main intent of the PR, right?

Reply:
Yes. This is something I noticed while testing from my desktop.
Is it better to address this in a separate PR?